### DISCUSSION TITLE
switch from contiv to NSM+Multus

### DISCUSSION THREAD

**Discussion by g-rabah** [ligato/vpp-agent#1801] [Q&A]
Hi everyone, 

I am rethinking my CNF based project build on contiv-vpp as the project is no longer maintained. I think the logical replacement would be multus + calico+NSM+Ligato. Have anyone here tried this kind of setup or have a tutorial i can use some help.  The learning curve is very steep. 

Thanks in advanced.

**Comment by chrismetz09** [comment]
Hi-
On the Ligato front, check out https://docs.ligato.io/en/latest/ and in particular, the quick start and tutorials. That will provide a decent overview of the VPP agent that manages the VPP data plane. NSM uses this architecture albeit without the KV data store.
